snuba/datasets/configuration/generic_metrics/storages/distributions_storage.yaml

```diff
@@ -0,0 +1,80 @@
+storage:
+  key: generic_metrics_distributions
+  set_key: generic_metrics_distributions
+
+schema:
+  columns:
+    [
+      { name: org_id, type: UInt, args: { size: 64 } },
+      { name: use_case_id, type: String },
+      { name: project_id, type: UInt, args: { size: 64 } },
+      { name: metric_id, type: UInt, args: { size: 64 } },
+      { name: timestamp, type: DateTime },
+      { name: retention_days, type: UInt, args: { size: 16 } },
+      {
+        name: tags,
+        type: Nested,
+        args:
+          [
```

Comment:
I think all argument lists that correspond to Python argument lists should be named for more straightforward forwards/backwards compatibility so this should be something like:

```yaml
args:
  subcolumns:
    (what you have now)
```

Reply:
Good catch, this is what I intended to do